packages/teleterm/src/sharedProcess/ptyHost/types.ts

```diff
@@ -0,0 +1,18 @@
+export type PtyProcessOptions = {
+  env: Record<string, string>;
+  path: string;
+  args: string[];
+  cwd?: string;
+  initCommand?: string;
+};
+
+export type PtyProcessType = {
```

Comment:
```suggestion
export type IPtyProcess = {
```

I see that we actually use this convention for `IAppContext`, perhaps we could use it here? I generally don't think we should have types called `*Type`, unless it's stuff like `*EventType`.